docs-chef-io/content/inspec/resources/group.md

```diff
@@ -13,6 +13,13 @@ platform = "os"
 
 Use the `group` Chef InSpec audit resource to test a single group on the system.
 
+Following system group is used in the `group` resource:
+
+- in **Non-Windows** system, resource only works with a group listed in file `/etc/group` and not a local group of the system.
+
+- in **Windows** system, resource only works with a local group of the system.
```

Comment:
I'm not sure if this is correct, but it would be good to define where or how the group policy is specified on Windows 

```suggestion
The `group` resource uses the following system groups:

- On **Non-Windows** systems the group resource tests a local group defined in the`/etc/group` file.
- On **Windows** systems the group resource tests a local group defined by Local Users and Groups.
```

Reply:
@IanMadd I think it sounds correct. @clintoncwolfe @Vasu1105 Any thoughts?

Reply:
I believe Ian's suggestion is correct.